tmt/result.py

```diff
@@ -542,6 +545,11 @@ def summary(results: list['Result']) -> str:
         if stats.get(ResultOutcome.ERROR):
             count, comment = fmf.utils.listed(stats[ResultOutcome.ERROR], 'error').split()
             comments.append(count + ' ' + click.style(comment, fg='magenta'))
+        if stats.get(ResultOutcome.PENDING):
+            count, comment = fmf.utils.listed(
```

Comment:
Actually, `fmf.util.listed` is used above to pick plural/singular, here a simple assign should be just enough:

```suggestion
            count, comment = stats[ResultOutcome.PENDING], 'pending'
```

Reply:
Thanks!